platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -1243,11 +1256,15 @@ def set_lpmode(self, lpmode):
                 # LPMode is active high; set or clear the bit accordingly
                 if lpmode is True:
                     reg_value = reg_value | mask
+                    write_val = 0x10
                 else:
                     reg_value = reg_value & ~mask
+                    write_val = 0x0
 
                 # Convert our register value back to a hex string and write back
                 self.pci_set_value(self.BASE_RES_PATH, reg_value, port_offset)
```

Comment:
To enable low power mode via ForceLowPwr bit, writing to LP mode hardware signal is NOOP at line 1265

![image](https://user-images.githubusercontent.com/45705344/136229702-0ca9d2d0-1b72-4ed8-b9c8-dc23bb2ee63f.png)